pkg/apis/scheduling/validation/validation.go

```diff
@@ -52,5 +52,9 @@ func ValidatePriorityClassUpdate(pc, oldPc *scheduling.PriorityClass) field.Erro
 	if pc.Value != oldPc.Value {
 		allErrs = append(allErrs, field.Forbidden(field.NewPath("Value"), "may not be changed in an update."))
 	}
+	// PreemptionPolicy is immutable and is checked by the ObjectMeta validator.
```

Comment:
also need to add validation to ValidatePriorityClass that if a non-nil value is specified, it is a valid preemption policy

Reply:
Done